src/index.html

```diff
@@ -1,6 +1,6 @@
 <!doctype html>
 <html lang="en">
-  <head>
+  <head class="html">
```

Comment:
The `<head>` tag should not have a `class` attribute. This is not standard practice and should be removed.